src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs

```diff
@@ -24,7 +26,7 @@ public RabbitMQClusterTransport(Topology topology, string connectionString, Queu
         /// <param name="port">The port of the node.</param>
         public void AddClusterNode(string host, int port = 5672)
```

Comment:
could we use `-1` as the default port instead? According to https://rabbitmq.github.io/rabbitmq-dotnet-client/api/RabbitMQ.Client.AmqpTcpEndpoint.html -1 represents the default port, which might be switch correctly depending on the SSL options?

Reply:
Updated.